store.css

```diff
@@ -0,0 +1,196 @@
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/*header */
+.header {
+  display: flex;
+  width: 100%;
+  height: 120px;
+  background-color: rgb(233, 233, 233);
+}
+
+/*header_logo*/
+.header_logo {
+  height: 75px;
+  margin-left: 150px;
+  top: -25px;
+  position: relative;
+  top: 25px;
+}
+
+/*Top Right Navigation Links*/
+a:hover {
+  color: orange;
+}
+
+.navigation__list {
+  display: flex;
+  position: relative;
+  bottom: 25px;
+  margin-left: 750px;
+  top: 30px;
+}
+
+.navigation__item {
+  color: silver;
+  padding: 0;
+  display: flex;
+}
+
+.navigation__link {
+  color: var(--grey-dark);
+  font-weight: 600;
+  text-decoration: none;
+  margin: 30px;
+}
+
+/*first section order page*/
+.section-1 {
+  display: flex;
+  margin-left: 150px;
+  margin-right: 900px;
+  position: relative;
+  top: 50px;
+  color: orangered;
+}
+
+.head_title {
+  /*display: flex;*/
+  color: orangered;
+}
+
+.form_details {
+  display: flex;
+  justify-content: space-between;
+  top: 25px;
+}
+.align_form {
+  display: flex;
+  flex-direction: column;
+}
+
+.address_details {
+  display: flex;
+  flex-direction: column;
+}
+
+fieldset {
+  border: none;
+  position: relative;
+  top: 50px;
+  margin-left: 150px;
+  margin-right: 1300px;
+}
+input {
+  margin: -10px;
+}
+
+/*image*/
+.section-2 {
+  display: flex;
+  margin-left: 1020px;
+  background-color: magenta;
+  margin-top: -650px;
+  position: relative;
+}
+/*working at desk image*/
+.working_desk {
+  width: 1000px;
+  height: 900px;
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+/* Radio Buttons */
+.radio_select label {
+  margin-right: 50px;
+  margin-left: 20px;
+}
+.radio_select input {
+  width: 20px;
+  height: 15px;
+  margin-left: -3px;
+}
+
+/* Radio Button Position*/
+.radio_select {
+  position: relative;
+  top: 90px;
+  margin-left: 150px;
+}
+
+/* Checkbox */
+.terms {
+  position: relative;
+  margin-left: 150px;
+  bottom: -125px;
+}
+
+.terms input {
+  margin: 5px;
+  margin-left: 2px;
+}
+
+.terms a {
+  color: orangered;
+}
+
+/*Place Order Button*/
+.button {
+  position: relative;
+  margin-left: 150px;
+  bottom: -200px;
+  color: white;
+  background-color: orangered;
+  border: none;
+  border-radius: 5px;
+  padding: 20px 40px;
+}
+
+/*footer*/
+.footer_start {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  position: relative;
+  height: 100px;
+  margin-top: 100px;
+}
+
+/* Join Us On */
+.footer_heading {
+  text-align: center;
+  font-size: 15px;
+  color: gray;
+  letter-spacing: 0.1rem;
+  position: relative;
+  bottom: 75px;
+}
+
+/*  2nd Section Images */
+.media_icons {
+  display: flex;
+  flex-direction: row;
+  max-height: 20px;
+  max-width: 20px;
+  margin-left: 425px;
+  margin-right: -350px;
+  position: relative;
+  bottom: 30px;
```

Comment:
And remove this unnecessary fixed position as well, it's not needed. 

To create space inside a block, add padding to it.
To move other items away from it, add margin.